compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -0,0 +1,1282 @@
+use std::{borrow::Cow, collections::hash_map::Entry, rc::Rc};
+
+use acvm::FieldElement;
+use im::Vector;
+use iter_extended::{try_vecmap, vecmap};
+use noirc_errors::Location;
+use rustc_hash::{FxHashMap, FxHashSet};
```

Comment:
```suggestion
use rustc_hash::{FxHashMap as HashMap, FxHashSet};
```
Let's keep with what we do throughout the codebase here

Reply:
We can then probably do the same thing for `FxHashSet`